ros2/node/nns_rclcpp_publisher.cc

```diff
@@ -126,15 +145,91 @@ gboolean NnsRclCppPublisher::publish (const guint num_tensors,
     rclcpp::spin_some(this->node_);
   }
 
-  return TRUE;
+  sq_writer = this->getRosBagWriter ();
+  if (sq_writer) {
```

Comment:
As long as this is actiaved (not null) only when RosBag is used, it's good. (please let me know)

Reply:
@myungjoo That is my intention and I checked this is activated only if the RosBag feature is explicitly set.